### PR TITLE
Added extensions to get paths of documents directory.

### PR DESCRIPTION
**Pull request by vilapuigvila**
## Checklist

- [ ] New Extension
- [ ] New Test
- [x] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by lfarah** [comment]
Hey @vilapuigvila, can you add Tests for this? Let us know if you have any questions!

Thanks for contributing! 🎉 👍

**Comment by lfarah** [comment]
@piv199 is this related to #163?

**Comment by vilapuigvila** [comment]
Ok, I'm going to try to do it.
Thanks for your support.

File: Sources/UIDeviceExtensions.swift [hunk 1]
```diff
@@ -85,6 +85,18 @@ extension UIDevice {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
 
+    /// EZSE: Returns path of documents directory
```
Comment (Khalian) [inline Sources/UIDeviceExtensions.swift]: These should not be in UIDevice.

**Comment by piv199** [comment]
@lfarah yes it resolves #163 

But I especially not know how it should be in this repo. Personally I use third-party library for this purposes if I am working much with directories.

**Comment by lfarah** [comment]
@vilapuigvila can you tell us about your experience using this and how this helped you?

**Comment by vilapuigvila** [comment]
Hi there! I'm sorry but last week I was very busy. I used to know where is the documents directory path for a different things, if I want to open core data xcmodel with external tool I need the path of it, in appDelegate always I have a log of documents directory path. Since ios8 documents directory path change every run and is this method is very helpful for me.

**Pull request closed without merging**